### ISSUE TITLE
Duplicate of (VSCode Won't Quit - Icon is Gone, Application Still Open #46255)

### ISSUE THREAD

**Issue by ChuyTheJesus** [Microsoft/vscode#62674]
I found a duplicate issue but the issue was locked by VSCode bot to collaborators only. 

- VSCode Version: 1.23.1
- OS Version:  MacOS 10.13.5

Steps to Reproduce:

1. Open VS Code and leave open for a few days....
2. VS Code crashes, right-click force-quit the application
3. The black VS Code Window is still open on desktop, with nothing displaying in the menu but a black background, but the icon is not marked in the taskbar below.
4. The "Code" menu and other menus show at the top of the screen when you click on the frozen window, however nothing is clickable due to a frozen screen.

<img width="1219" alt="screen shot 2018-11-06 at 11 40 27 am" src="https://user-images.githubusercontent.com/30666169/48082706-e2cb3380-e1b8-11e8-9819-f9a6103b0f8d.png">

Does this issue occur when all extensions are disabled?: Yes/No
Unsure, still trying to figure out how to shut it down without rebooting my laptop. I also don't know how to test this again.

Does this issue occur when all extensions are disabled?: Yes/No

**Comment by chrmarti** [comment]
Please follow https://github.com/Microsoft/vscode/wiki/Native-Crash-Issues.

**Comment by ChuyTheJesus** [comment]
Update: I wasn't able to close out the window without rebooting my laptop. After rebooting my laptop, the window was gone, and VSCode worked normally. I just hope this doesn't happen again as I lost some time of work, and had quite a few applications and windows open that were a hassle to close before rebooting my device.
 
Here is the crash report:


[ReportCrash_2018-11-06-091946_Chuy-Perez.crash.zip](https://github.com/Microsoft/vscode/files/2558835/ReportCrash_2018-11-06-091946_Chuy-Perez.crash.zip)

**Comment by chrmarti** [comment]
@ChuyTheJesus I missed that you are on an older version of VS Code. Could you update and let us know if you see this again with the latest version? Thanks.

**Comment by ChuyTheJesus** [comment]
Understood, will do.

How long will this ticket remain open for me to update if it happens or not? Or will I still be able to add comments to it after being closed?

**Comment by ChuyTheJesus** [comment]
Scratch that, just answered my own question. 
Thanks

**Comment by chrmarti** [comment]
It will lock only after 45 days of inactivity. You can always create a new issue after that and refer to this one.
